project/project/plugins.sbt

```diff
@@ -1,3 +1,3 @@
-val ScalafmtVersion = "1.7"
+val ScalafmtVersion = "1.7" // https://github.com/lucidsoftware/neo-sbt-scalafmt/issues/31
```

Comment:
In theory, one could write a bit of sbt DSL code that during the meta-build of `/project/project`, it **writes** a file with these variables, and then the main build under `/project` **reads** the generated file.

I looked at a couple of stack-overflow [options](https://stackoverflow.com/questions/23944108/how-to-share-version-values-between-project-plugins-sbt-and-project-build-scala) and hated all of the suggestions. Possibly could be done in a future ticket though?